### PR TITLE
ref(ts): Convert <SidebarOrgSummary> to typescript

### PR DESCRIPTION
**Pull request by matejminar**

### CODE REVIEW

**Review by dashed — APPROVED** [review]
Looks good to me 👍

File: src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx [hunk 1]
```diff
@@ -4,35 +4,36 @@ import styled from 'react-emotion';
...
+const SidebarOrgSummary: React.FC<Props> = ({organization}) => {
+  const projects = organization.projects && organization.projects.length;
+  const extra: string[] = [];
 
-    if (projects) {
-      extra.push(`${projects} projects`);
-    }
+  if (projects) {
+    // should we think about pluralization here?
+    extra.push(`${projects} projects`);
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx]: We likely should use `tn()` in the locale module to translation and pluralization.

Comment (matejminar) [reply src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx]: used it, thanks 👍

File: src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx [hunk 2]
```diff
@@ -4,35 +4,36 @@ import styled from 'react-emotion';
...
+  }
 
-    return (
-      <OrgSummary>
-        <OrganizationAvatar css={{flexShrink: 0}} organization={organization} size={36} />
```
[the remark this answers is not in the document]
Comment (matejminar) [reply src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx]: That's what I thought, but after inspection, I found out that the css prop was not being applied here, that's why I changed it to style, which is according to underlying components' types being passed down and applied to the final element. But anyway, the  final element (`StyledBaseAvatar` in `app/components/avatar/baseAvatar`) has `flex-shrink: 0` by default, so do you think we can keep it removed altogether?

[the remark this answers is not in the document]
Comment (billyvg) [reply src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx]: It does look like it's being applied, this is from production (I believe it's this component): 
image

[the remark this answers is not in the document]
Comment (matejminar) [reply src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx]: Yes, it should be that component, but it's `flex-shrink` comes imho from: https://github.com/getsentry/sentry/blob/893720d456c71258c276e0cb700e9a6e10870e19/src/sentry/static/sentry/app/components/avatar/baseAvatar.tsx#L241

[the remark this answers is not in the document]
Comment (billyvg) [reply src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx]: Ah, cool, let's kill it then

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (matejminar) [reply src/sentry/static/sentry/app/components/sidebar/sidebarOrgSummary.tsx]: awesome, thank you

**Pull request merged**